features/application/Loading.feature

```diff
@@ -19,21 +19,16 @@ Feature: index
     And I wait for 2000 milliseconds
     Then the "$loading" element doesn't exist
     And the "$logo" element exists
-
-  # TODO: This test needs fixing it currently console.errors
-  @skip
```

Comment:
Another benefit here is that this test now passes, and whilst it did also pass in its previous incarnation in the CLI tests, when I tried to recreate it in a real browser it failed.